### ISSUE TITLE
iOS App - touch and hold gesture or 3D Touch to Create notes

### ISSUE THREAD

**Issue by dernorberto** [standardnotes/forum#991]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes.

**Is your feature request related to a problem? Please describe.**
This feature requests relates to adding (not replacing) an even quicker way to create notes.

**Describe the solution you'd like**
iOS supports a long-press/3d-touch triggered context menu on the app icon. With a "New Note" shortcut it would then open the app in a blank note and ready to go. This feature exists in e.g. the mail app, wunderlist, todoist, etc...

**Describe alternatives you've considered**
The only alternative is the default way of opening app and clicking the + button.

**Additional context**
N/A

**Comment by joshcsullivan** [comment]
+1 
I would love to see this feature get implemented. Quick creation of notes is essential, and this would speed up the process drastically.

**Comment by cyb3rz3us** [comment]
I agree that this is a very valuable feature.  With that in mind, might I also add to this to request the creation of an iOS widget?

 I know there is already a widget FR for Android but I think all SN mobile devices would benefit from both widgets and the FR outlined herein.

**Comment by JaspalSuri** [comment]
Hi, thanks for the feedback, everyone! As an iOS user, I would love to see this feature as well. I will compile it for review by our mobile dev (@radko93 ). :)
